src/Schema/Events/ImpressionTracking.ts

```diff
@@ -82,6 +82,28 @@ export interface TooltipViewed {
   type: string
 }
 
+/**
+ * A user viewed a tooltip. `type` will describe what tooltip it is (follow, edit-follows, etc.)
+ *
+ * This schema describes events sent to Segment from [[tooltipViewed]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tooltipViewed",
+ *    context_module: "newWorksForYouRail",
+ *    context_owner_type: "home",
+ *    type: "long-press-artwork-context-menu"
+ *  }
+ * ```
+ */
+export interface TooltipViewedApp {
```

Comment:
I don't see the prefix `App` being used. Maybe we can just remove it 🤔 And because this is related to Onboarding, maybe we should add it to the name as well.

Reply:
`tooltipViewed` interface has some params that we don't eed to mention explicitly on eigen, also, it uses type PageOwnerType which is web specific.
I will rename the `TooltipViewedApp` interface